### PR TITLE
New photo from Laurence :)

### PR DESCRIPTION
**Pull request by laurence-hubbard**

### CODE REVIEW

**Comment by laurence-hubbard** [comment]
Hope googleusercontent is ok :)

File: src/config/images.json [hunk 1]
```diff
@@ -5,6 +5,11 @@
   {
     "src": "https://farm1.staticflickr.com/853/43086898084_61e7456585_o.jpg"
   },
+  {
+    "src": "https://lh3.googleusercontent.com/OJH9JpcjbGNHdrdzVr_4-6R1bdOT79nhlyJDX78Sk3aryLHqymTx_Jln2fc3OiNBTDOTvICfq_MxOVwRyUW7QTEbEmqgtPRaHjSBznlOHJU0TJ2a1ldUxmFg5CHbRJ2yzv5Jqjpdyx8Zr7TCZzFNFx31VosI8sfEANeo33LYXp1PrqLvXDZA8QYGkR5Ep-TV4fdN_0sosdI12qTziZO4NVHUlpd07flnHATJ5NKJ4aS5z2s67aNTsqW8SQUw5GR3l9QSlUxLJRw4fyBzlxIiu_jcSVJMSliQSfrpFOLtGOB3x7zE8ShPzpdUGDjGodAQcOJTLupSHGr3XiwjAQ9Cjzmo2Yyhc6JXwCps_GwSi4fRccuj71dFhQGwTZE1vs5CWc0oke9ssOwo2K0AAhUQTNizBwcF-AvgT-3h_iQEEz8CklDRXZMZYV9p-F55xgUcSOXv9kkL0gq3tv6gb6pBv9FwTMGh1TI4yrfiB3lxa2d7p98hP0GNTiOzEEjzAygr6dab6SUA0lzxvkMa-hsJgVkgR8C6AU3W-WN5mZM5ydUsTm5od9ZTH7gP-HbJs5nbVKs1mkxCBmOJFuAxI_Og1xnVhhwRRjLOyAJn3Zzj=w1948-h1460-no",
```
Comment (rkclark) [inline src/config/images.json:9]: If I follow this link it doesn't resolve to an image! Does it for you?

Comment (laurence-hubbard) [reply src/config/images.json:9]: Hmmm it did yesterday! Let me figure this out and get a new link

**Comment by rkclark** [comment]
closing for now

**Pull request closed without merging**